src/Features/Core/Portable/EditAndContinue/EmitSolutionUpdateResults.cs

```diff
@@ -149,7 +149,7 @@ public async Task<ImmutableArray<Diagnostic>> GetAllDiagnosticsAsync(Solution so
                     continue;
                 }
 
-                var fileSpan = data.DataLocation.GetFileLinePositionSpan();
+                var fileSpan = data.DataLocation.GetNormalizedFilePathLinePositionSpan();
```

Comment:
i renamed this (and doc'ed it) to be clearer about the semantics.  Namely that you always get a normalized ('full') file path back.  As opposed to directly interacting with the 'mapped file span' which just contains a file-name that is intended to be resolved against the full path of hte 'original' diagnostic span.